src/NHibernate.Test/NHSpecificTest/GH3363/FixtureByCode.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System.Linq;
+using NHibernate.Cfg.MappingSchema;
+using NHibernate.Mapping.ByCode;
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.GH3363
```

Comment:
NH is for the old Jira bug tracker. Use GH for GitHub issues.

Reply:
I should have done

Reply:
I also removed useless test I added in advance to show working cases, only single failing test is now in the PR

Reply:
@fredericDelaporte sorry didn't noticed your edit I thought was some commit from async gen and just forced. Sorry I didn't mean to be rude just n00b